src/GoblinList.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import Goblin from './Goblin';
 
-export default function GoblinList(props) {
+export default function GoblinList({ handleDeleteGoblin, goblins }) {
   return (
     <div className='goblin-list quarter'>
-      {/* map over your goblins and render out a Goblin component for each goblin. You've seen this before. The only difference here is that you need to pass handleDeleteGoblin (a prop that is a function), as well */}
+      {
+        goblins.map((goblin, i) => <div key={goblin.name + i}> (
```

Comment:
currently, the parenthesis around the Goblin component are causing parenthesis to render to the page for each new goblin